src/server/create.ts

```diff
@@ -23,6 +23,9 @@
  --------------
  ******/
 
+// workaround for lack of typescript types for mojaloop dependencies
+// eslint-disable-next-line
+/// <reference path="../../ambient.d.ts"/>
 import { validateRoutes } from '@mojaloop/central-services-error-handling'
```

Comment:
maybe it is worth adding proper declarations to ambient.d.ts... here is example: https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/ambient.d.ts#L32

so you can extend and add a proper definition for `validateRoutes`....

Reply:
I believe they're defined here in https://github.com/mojaloop/als-consent-oracle/blob/master/ambient.d.ts

but for some reason I get
`src/server/create.ts:26:32 - error TS7016: Could not find a declaration file for module '@mojaloop/central-services-error-handling'` if the ///<reference path... is missing from this file.

Not sure if the declaration is wrong or the config is wrong but I had removed this reference path when copying over from auth-service only to realize later that is breaks if you try to run the server.
https://github.com/mojaloop/auth-service/blob/master/src/server/create.ts#L26